infra/main.parameters.json

```
@@ -158,6 +158,9 @@
     "defaultModel": {
       "value": "${DEFAULT_MODEL}"
     },
+    "useInjectionProtection": {
+      "value": "${USE_INJECTION_PROTECTION}"
```

Let's also add the default here!

```suggestion
      "value": "${USE_INJECTION_PROTECTION=false}"
```

---

Done